util/Migrator/DbScripts/2020_12_04_00_OrgUserReadByOrgEmail.sql

```diff
@@ -0,0 +1,24 @@
+IF OBJECT_ID('[dbo].[OrganizationUser_ReadByOrganizationEmail]') IS NOT NULL
```

Comment:
@vincentsalucci, This check and `DROP` applies to `OrganizationUser_ReadByOrganizationEmail`; however the `CREATE` below refers to `OrganizationUser_ReadByOrganizationIdEmail` which is different.

Just noticed this as it's breaking my deployment so we'll want to get this corrected in a new PR.